lib/options.js

```diff
@@ -13,11 +13,14 @@ export type OptionsFormat = {
  * The plugin options
  */
 class Options extends Record({
+    maxDepth: 12,
```

Comment:
options should be humanly logical. So they should contain settings as programmers think about them and maybe change them for machines laters inside